### PR TITLE
Feature/pybind update

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: resources/stylesheets/inviwo.qss [hunk 1]
```diff
@@ -82,21 +82,11 @@ QToolBar
 QToolBar {
     spacing: 0.1875em;
     padding: 0.0625em;
-    border: none;
-    border-bottom: 0.125em solid #2b2b2d;
-    padding-right: 0.5em;
     background-color: #323235;
-}
-QToolBar::handle {
```
Comment (martinfalk) [inline resources/stylesheets/inviwo.qss:90]: why were those removed?
image

**Review by martinfalk — APPROVED** [review]

File: CMakePresets.json [hunk 2]
```diff
@@ -67,18 +68,16 @@
 
     {
       "name": "msvc-dev",
-      "displayName": "MSVC Development",
-      "description": "MSVC Development configuration",
+      "displayName": "MSVC 2022",
+      "description": "MSVC 2022 Development configuration",
```
Comment (ResearchDaniel) [inline CMakePresets.json:72]:
```suggestion
      "description": "Visual Studio 2022 setup suitable for development",
```

File: CMakePresets.json [hunk 3]
```diff
@@ -67,18 +68,16 @@
...
-      "inherits" : ["modules", "develop"]
+      "inherits" : ["develop"]
     },
     {
       "name": "msvc-dev-vcpkg",
-      "displayName": "MSVC Development VCPKG",
-      "description": "MSVC Development VCPKG configuration",
-      "binaryDir": "${sourceParentDir}/builds/msvc-dev-vcpkg",
+      "displayName": "MSVC 2022 Using vcpkg",
+      "description": "MSVC 2022 Development using vcpkg configuration",
```
Comment (ResearchDaniel) [inline CMakePresets.json:80]:
```suggestion
      "description": "Visual Studio 2022 development using vcpkg package manager",
```

File: CMakePresets.json (hunk 2 above)
Comment (ResearchDaniel) [reply CMakePresets.json:72]: I would prefer if the "development" config could be stripped down a bit to provide the most essential components by excluding the minimal apps. We might want to separate "Core development" and "Module development", where "Module development" would be more light-weight.

If we could define what "development" means that would also be nice to have in the description. This might not be the main purpose of this PR though.

**Review by ResearchDaniel — APPROVED** [review]

File: resources/stylesheets/inviwo.qss (hunk 1 above)
Comment (petersteneteg) [reply resources/stylesheets/inviwo.qss:90]: I made them non movable, and non floatable. then that bar is not visible...

File: CMakePresets.json (hunk 3 above)
Comment (petersteneteg) [reply CMakePresets.json:80]: I can try, incorporate this in a future PR
